zadanie_domowe.sql

```diff
@@ -13,73 +14,91 @@ CREATE TABLE `tabela_pracownikow` (
   
 -- Pobiera wszystkich pracowników i wyświetla ich w kolejności alfabetycznej po nazwisku
-INSERT INTO `zadanie_domowe`.`tabela_pracownikow` (`first_name`, `last_name`, `salary`, `birth_day`, `department`) VALUES ('Krzychu', 'Przystarz', '4999.99', '1976-03-24', 'Junior Java Developer');
-INSERT INTO `zadanie_domowe`.`tabela_pracownikow` (`first_name`, `last_name`, `salary`, `birth_day`, `department`) VALUES ('Maciej', 'Przystarz', '5499', '2000-06-20', 'Junior Java Developer');
-INSERT INTO `zadanie_domowe`.`tabela_pracownikow` (`first_name`, `last_name`, `salary`, `birth_day`, `department`) VALUES ('Marcin', 'Kunert', '11999', '1994-01.01', 'Senior Java Developer');
-INSERT INTO `zadanie_domowe`.`tabela_pracownikow` (`first_name`, `last_name`, `salary`, `birth_day`, `department`) VALUES ('Nati', 'Koltko', '9999', '1998-12-13', 'Scrum Master');
-INSERT INTO `zadanie_domowe`.`tabela_pracownikow` (`first_name`, `last_name`, `salary`, `birth_day`, `department`) VALUES ('Sławek', 'Ludwiczak', '13999', '1994-02-02', 'Senior Java Developer');
+INSERT INTO `zadanie_domowe`.`pracownicy` (`first_name`, `last_name`, `salary`, `birth_day`, `department`) VALUES ('Krzychu', 'Przystarz', '4999.99', '1993-05-13', 'Junior Java Developer');
+INSERT INTO `zadanie_domowe`.`pracownicy` (`first_name`, `last_name`, `salary`, `birth_day`, `department`) VALUES ('Maciej', 'Przystarz', '5499', '1993-05-14', 'Junior Java Developer');
+INSERT INTO `zadanie_domowe`.`pracownicy` (`first_name`, `last_name`, `salary`, `birth_day`, `department`) VALUES ('Marcin', 'Kunert', '11999', '1994-01.01', 'Senior Java Developer');
+INSERT INTO `zadanie_domowe`.`pracownicy` (`first_name`, `last_name`, `salary`, `birth_day`, `department`) VALUES ('Nati', 'Koltko', '9999', '1998-12-13', 'Scrum Master');
+INSERT INTO `zadanie_domowe`.`pracownicy` (`first_name`, `last_name`, `salary`, `birth_day`, `department`) VALUES ('Sławek', 'Ludwiczak', '13999', '1994-02-02', 'Senior Java Developer');
 
 -- Pobiera pracowników na wybranym stanowisku
-SELECT * FROM tabela_pracownikow ORDER BY last_name;
+SELECT * FROM pracownicy ORDER BY last_name;
 
 -- Pobiera pracowników, którzy mają co najmniej 30 lat
+SELECT * FROM pracownicy WHERE birth_day <= DATE_SUB(CURDATE(), INTERVAL 30 YEAR);
 
-SELECT * FROM tabela_pracownikow WHERE birth_day <= '1993-01-01';
 -- Zwiększa wypłatę pracowników na wybranym stanowisku o 10%
 SET SQL_SAFE_UPDATES = 0;
-UPDATE tabela_pracownikow SET salary = salary * 1.1 WHERE department = 'Scrum Master';
+UPDATE pracownicy SET salary = salary * 1.1 WHERE department = 'Scrum Master';
+
 -- Pobiera najmłodszego z pracowników (uwzględnij przypadek, że może być kilku urodzonych tego samego dnia)
-SELECT * FROM tabela_pracownikow WHERE birth_day = (SELECT max(birth_day) FROM tabela_pracownikow);
+SELECT * FROM pracownicy WHERE birth_day = (SELECT max(birth_day) FROM pracownicy);
+
 -- Usuwa tabelę pracownik
-DELETE FROM tabela_pracownikow;
+DELETE FROM pracownicy;
```

Comment:
Usunięcie tabeli to drop, a nie delete. Delete powoduje usunięcie zawartości i pozostawia tabelę, drop usuwa całą tabelę